packages/lodestar/src/api/impl/beacon/fork.ts

```diff
@@ -0,0 +1,13 @@
+import { Fork, uint64, BeaconState } from "@chainsafe/eth2.0-types";
+import { IBeaconDb } from "../../../db";
+import { IBeaconChain } from "../../../chain";
+
+
+export async function getFork(db: IBeaconDb, chain: IBeaconChain): Promise<{fork: Fork; chainId: uint64}> {
+  const state: BeaconState = await db.state.getLatest();
```

Comment:
niot sure if api specifies error in case chain didn't yet started (getLatest returns null)?